internal/common/common.go

```diff
@@ -434,3 +434,7 @@ func ApplyYamlPatch(src []byte, ops []byte) ([]byte, error) {
 
 	return patched, nil
 }
+
+func VipVerificationAddress(v models.VipVerification) *models.VipVerification {
```

Comment:
For consistency, can we move it to internal/common/conversions.go, and call it VipVerificationPtr.
We already have some similar funcs there.

Reply:
OK